src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/SubmissionDispatcher.java

```diff
@@ -31,7 +33,7 @@ public SubmissionDispatcher(Catalog imageStore) {
 	}
 
 	public SubmissionDispatcher(Properties properties) throws SQLException {
-		this.imageStore = new JDBCImageDataStore(properties);
+		this.imageStore = new JDBCCatalog(properties);
```

Comment:
why are we still using `imageStore`? please, rename to `this.catalog`

Reply:
Done